doc/ref_model/README.md

```diff
@@ -48,5 +48,5 @@
 * [Chapter 08 - Compliance, Verification, and Certification](chapters/chapter08.md)
 * [Chapter 09 - Life Cycle Management](chapters/chapter09.md)
 * [Chapter 10 - Challenges and Gaps](chapters/chapter10.md)
-* [Appendix A - Guidelines For VNF Vendors](chapters/appendix-a.md)
+* [Appendix A - Guidelines For Application Vendors](chapters/appendix-a.md)
 * [Glossary](../tech/glossary.md)
```

Comment:
@EditKoselak lets use Workload as we don't use application

Reply:
> @EditKoselak lets use Workload as we don't use application

One of the related corrections, #1539 has been merged into the master, and we use Application there. Please advise on how to proceed.

Reply:
@EditKoselak thanks to figure it out 
I think workload is common use terminology 
@rabi-abdel please advice

Reply:
M3 milestone, for tomorrow, is for contribution changes.  The M4 milestone's "goal is to perform the final proof reading of the document before it is released. "

@rabi-abdel Can we consider this as item that could be cleared up during proof-reading?

Reply:
@kedmison @ASawwaf we will inform @bfcohen to make sure she edit those. for this PR, I think it will be OK to approve as is.